index.d.ts

```diff
@@ -405,4 +405,6 @@ declare module 'plaid' {
     [env: string]: string;
   }
   const environments: PlaidEnvironments;
+
+  export function isPlaidError(err: Error | Object): err is Object;
```

Comment:
I think we need to something like `err is PlaidError` in order to typescript callers to benefit from this user defined type guard.